src/styles/blocks/_page.scss

```
@@ -0,0 +1,7 @@
+.page {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
```

Styles are good here, it is also a good practice to sort css properties so the code is cleaner and easier to ready. Also you dont need to set 100% width. By default, the <body> element in HTML will take up the entire width of the viewport, which is the visible area of the web page in the browser window. The default width of block-level elements in HTML is 100% of the parent container's width
```suggestion
.page {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
}
```